content/shmem_team_create_ctx.tex

```diff
@@ -61,7 +61,7 @@
 \begin{apiexamples}
     \apicexample
     {The following example demonstrates the use of contexts for multiple teams in a
-    \Cstd[11] program. This example shows contexts being used to communicate within
+    \CorCpp program. This example shows contexts being used to communicate within
```

Comment:
Do the examples use any of the C11 generic selection APIs?

Reply:
Not that I saw - only the C/C++ APIs.

Reply:
Thanks for confirming